pkg/localize/locales/en/cmd/kafka_delete.en.toml

```diff
@@ -51,4 +47,4 @@ one = 'Deleting Kafka instance'
 
 [kafka.delete.log.info.deleteSuccess]
 description = 'Info message when instance was deleted'
-one = 'Kafka instance "{{.Name}}" has successfully been deleted'
\ No newline at end of file
+one = 'Kafka instance "{{.Name}}" is being deleted. You can monitor its progress by running "rhoas status".'
```

Comment:
We should also expect to see the same message when skipping confirmation.

Currently it shows:
```shell
❯ rhoas kafka delete -y
Deleting Kafka instance "enda-kafka"

Kafka instance "enda-kafka" has successfully been deleted
```

Reply:
Updated
![Screenshot from 2021-06-03 19-50-43](https://user-images.githubusercontent.com/23582438/120660522-08ba3100-c4a5-11eb-9015-e638968f5c09.png)